pkgs/applications/networking/giara/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitLab, lib, pkgs }:
+
+let
+  pname = "giara";
+  version = "0.1.1";
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "GabMus";
+    repo = pname;
+    rev = version;
+    sha256 = "038isdbvax3361kdi2rsgasyic7nmz0l9pd8r531bxfcsqcd1ny8";
+  };
+in
+
+stdenv.mkDerivation {
+  inherit pname version src;
+
+  nativeBuildInputs = with pkgs; [
+    meson pkg-config cmake ninja python3 python3Packages.wrapPython wrapGAppsHook
```

Comment:
Are you sure `cmake` is needed? Meson supports it for finding dependencies but typically, `pkg-config` will be sufficient.